piety-promoted-v4/modernized/07-chapter-07.adoc

```diff
@@ -99,7 +104,10 @@ her mind was preserved in great watchfulness and care on her own account,
 and in a lively concern for others,
 as appears by the following extracts from some of her letters and memorandums.
 
-LYONS, 1816.--Tenth month, 31. "`Our separation from the church,
+[.embedded-content-document]
```

Comment:
Somehow this [.embedded-content-document] wasn't labeled a letter (or anything else). Also... I think it would look a little better if the city names (Lyons, Nice) just had the first letter capitalized, and not be in all caps. I also happened to notice that the third excerpt here is missing the long dash between 1817. and First month (just to make it look like the other ones. 👍🏻